vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/SlideManagerTest.java

```diff
@@ -0,0 +1,143 @@
+        moduleId = "MOD000000002";
+        sequenceId = "SEQ000000004";
+        sequenceIdOther = "SEQ000000005";
+        slidesList.get(0).setId(slideId);
+    }
+
+    @Test
+    public void test_getSlideSequence_slideExistsInSequence() {
+
+        // Positive scenario - Slide present in Sequence
+
+        sequenceObj.setId(sequenceId);
+        sequenceObj.setSlides(slidesList);
+        List<Sequence> sequencesList = new ArrayList<>();
+        sequencesList.add(sequenceObj);
+        Mockito.when(sequenceRepo.findSequencesForModule(moduleId)).thenReturn(sequencesList);
+        List<Sequence> slideSequencePresent = slideManagerToTest.getSlideSequences(slideId, moduleId);
+
+        String actualSlideIdSequence = slideSequencePresent.get(0).getSlides().get(0).getId();
+        String expectedSlideIdSequence = sequencesList.get(0).getSlides().get(0).getId();
+
+        Assert.assertEquals(actualSlideIdSequence, expectedSlideIdSequence);
+        Assert.assertEquals(slideSequencePresent.size(), sequencesList.size());
+
+    }
+
+    @Test
+    public void test_getSlideSequence_slideNotExistsInSequence() {
+
+        sequenceObj.setId(sequenceId);
+        sequenceObj.setSlides(slidesList);
+        List<Sequence> sequencesList = new ArrayList<>();
+        sequencesList.add(sequenceObj);
+        Mockito.when(sequenceRepo.findSequencesForModule(moduleId)).thenReturn(sequencesList);
+
+        List<Sequence> actualSequenceSlideListNotPresent = slideManagerToTest.getSlideSequences(slideIdNotInSequence,
+                moduleId);
+        Assert.assertTrue(actualSequenceSlideListNotPresent.isEmpty());
+    }
+
+    @Test
+    public void test_deleteSlideById_slideIdPresent() throws SlideDoesNotExistException {
+
+        sequenceObj.setId(sequenceId);
+        sequenceObj.setSlides(slidesList);
+        List<Sequence> sequencesList = new ArrayList<>();
+        sequencesList.add(sequenceObj);
+        Mockito.when(sequenceRepo.findSequencesForModule(moduleId)).thenReturn(sequencesList);
+
+        Slide slide = new Slide();
+        slide.setId(slideId);
+        Optional<Slide> slideObj = Optional.of(slide);
+        Mockito.when(slideRepo.findById(slide.getId())).thenReturn(slideObj);
+        slideManagerToTest.deleteSlideById(slideId, moduleId);
+        Mockito.verify(slideRepo).delete((Slide) slide);
+    }
+
+    @Test
+    public void test_deleteSlideById_slideIdNotPresent() throws SlideDoesNotExistException {
+
+        Mockito.when(slideRepo.findById(Mockito.anyString())).thenReturn(Optional.empty());
+
+        sequenceObj.setId(sequenceId);
+        sequenceObj.setSlides(slidesList);
+
+        List<Sequence> sequencesList = new ArrayList<>();
+        sequencesList.add(sequenceObj);
+        Mockito.when(sequenceRepo.findSequencesForModule(moduleId)).thenReturn(sequencesList);
+        slideManagerToTest.deleteSlideById(slideId, moduleId);
+        Mockito.doThrow(SlideDoesNotExistException.class).when(slideRepo).delete(null);

```

Comment:
a couple of things, first I"m pretty sure the repo doesn't throw a `SlideDoesNotExistException`. And second, you should not try to delete null. Check that first. Third, if you expect an exception to be thrown you need to have your test case expect that so the test passes.